src/server/scriptVersionCache.ts

```diff
@@ -632,6 +632,7 @@ export class LineNode implements LineCollection {
 
     walk(rangeStart: number, rangeLength: number, walkFns: LineIndexWalker): void {
         // assume (rangeStart < this.totalChars) && (rangeLength <= this.totalChars)
+        if (this.children.length === 0) return;

```

Comment:
Is `this.children` always defined?

Reply:
Yes, the type is `LineCollection[]` and the constructor defaults it to `[]` if it's not provided.